File: 백준/Bronze/1919. 애너그램 만들기/애너그램 만들기.java
import java.io.*;
import java.util.*;

public class Main {
    public static void main(String[] args) throws IOException {

        Scanner sc = new Scanner(System.in);
        String str1 = sc.nextLine();
        String str2 = sc.nextLine();

        int[] count1 = new int[26];
        int[] count2 = new int[26];

        for(char a: str1.toCharArray()){
            count1[a-'a']++;
        }
        for(char a: str2.toCharArray()){
            count2[a-'a']++;
        }
        int result = 0;
        for(int i = 0; i<26;i++){
            result+=Math.abs(count1[i]-count2[i]);
        }
        System.out.println(result);
    }
}

File: 백준/Bronze/1919. 애너그램 만들기/애너그램 만들기.py
str1 = input()
str2 = input()
arr1 = [0] * 26
arr2 = [0] * 26
for i in str1:
    arr1[ord(i)-97] += 1
for i in str2:
    arr2[ord(i)-97] += 1
result = 0
for i in range(26):
    result+=abs(arr1[i] - arr2[i])
print(result)